pallets/proxy/src/lib.rs

```diff
@@ -0,0 +1,213 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[cfg(test)]
+mod tests;
+#[cfg(test)]
+mod mock;
+
+/// Edit this file to define custom logic or remove it if it is not needed.
+/// Learn more about FRAME and the core library of Substrate FRAME pallets:
+/// <https://docs.substrate.io/v3/runtime/frame>
+pub use pallet::*;
+
+
+#[frame_support::pallet]
+pub mod pallet {
+	use frame_support::{dispatch::DispatchResultWithPostInfo, pallet_prelude::*, sp_runtime::traits::{StaticLookup},};
+	use frame_system::pallet_prelude::*;
+	use pallet_acurast::{JobRegistration, Script, Fulfillment, AttestationChain,
+						 CertificateRevocationListUpdate, AllowedSourcesUpdate};
+	use xcm::v2::{OriginKind, SendError, };
+	use xcm::v2::Instruction::{Transact, DescendOrigin};
+	use xcm::v2::{Junction::{Parachain, AccountId32}, SendXcm, Xcm, Junctions::{X1}};
+	use xcm::v2::prelude::*;
+	use frame_support::inherent::Vec;
+	// use bytes:str::ToBytes;
+
+	#[derive(Clone, PartialEq, Encode, Decode, RuntimeDebug, TypeInfo)]
+	#[allow(non_camel_case_types)]
+	pub enum ProxyCall<T: Config> {
+		#[codec(index = 0u8)]
+		register { registration: JobRegistration<T::AccountId, T::RegistrationExtra> },
+
+		#[codec(index = 1u8)]
+		deregister { script: Script },
+
+		#[codec(index = 2u8)]
+		update_allowed_sources { script: Script, updates: Vec<AllowedSourcesUpdate<T::AccountId>> },
+
+		#[codec(index = 3u8)]
+		fulfill { fulfillment: Fulfillment, requester: <T::Lookup as StaticLookup>::Source },
+
+		#[codec(index = 4u8)]
+		submit_attestation { attestation_chain: AttestationChain },
+
+		#[codec(index = 5u8)]
+		update_certificate_revocation_list { update: CertificateRevocationListUpdate }
```

Comment:
@JuaniRios we can remove these 2 from the proxy, they should be called directly

Reply:
done